src/rust/engine/README.md

```diff
@@ -1,24 +1,16 @@
 A native-code implementation of the pants v2 engine. See:
 
-    https://docs.google.com/document/d/1C64MreDeVoZAl3HrqtWUVE-qnj3MyWW0NQ52xeWw/edit
+    https://docs.google.com/document/d/1C64MreDeVoZAl3HrqtWUVE-qnj3MyWW0NQ52xejjaWw/edit?usp=sharing
 
-To build for development, run:
+To build for development, simply run pants from the root of the repo:
 
-    cargo build
+    ./pants list 3rdparty::
```

Comment:
I tend to `./pants -v` which is sufficient to trigger a build, or `./pants goals` which is sufficient to actually initialise the engine, but `./pants list 3rdparty::` is reasonable too :)